irc_bot.rb

```diff
@@ -1,18 +1,120 @@
 # customize the name of your bot on the following line
-@name = "Sarah-Bot"
+require_relative './irc_bot_methods.rb'
 
+@name = "EquineVetBot"
+
+@chosen_problem = :none
+@tree_level = 0
+@repetition_lameness = 0
+@horse_general_information
+@lameness_diagnostic_program = 0
+@lameness_diagnostic_state
 # This method gets called, whenever a message is sent to our IRC channel. In it you can react to
 # the users' inputs in whatever way you like...
+
+
+
+
+
 def handle_channel_message(message)
+  if message.include?("return")
+    @tree_level = 0
+    @chosen_problem = :none
+  end
+  if @tree_level == 0
+    p message.split[3]
+    irc_send("PRIVMSG #rubymonstas :Hello and welcome to the EquineVetBot. What is your emergency?")
+    irc_send("PRIVMSG #rubymonstas :I can help you with the following frequent issues in equine medicine: Colic, Lameness, Wounds ")
+    irc_send("PRIVMSG #rubymonstas :Enter your issue")
+    irc_send("PRIVMSG #rubymonstas :If at any point of the program you want to start over, enter return.")
+    @tree_level = 1
+  end
+
+
+  #colic
+  
+  if @tree_level ==1 && message.downcase.include?("colic")
+    @chosen_problem = :colic
```

Comment:
```suggestion
    @chosen_problem = :colic
    handle_message_colic(message)
    @tree_level = 2
```
Then delete the if-statement below :)